src/main/webapp/scripts/sys/isoFipsCountryMapScript.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * CU Generic ISO/FIPS Country modification
+ */
+const ISO_COUNTRY_CODE_FIELD = 'document.newMaintainableObject.isoCountryCode';
```

Comment:
If you're not using the ISO_COUNTRY_CODE_FIELD or FIPS_COUNTRY_CODE_FIELD constants, you could remove them.